calendar-helper/index.js

```diff
@@ -107,8 +132,13 @@ class CalendarHelper {
 		let date = dateNumber ? new Date(dateNumber) : inputDate;
 
 		let page = [];
+
+		if (this.options.shortcutButtons.length > 0) {
+			this.addShortcutButtons(page, m);
+		}
```

Comment:
Here you better add a null check:
```suggestion
		const shortcutButtons = this.options.shortcutButtons
		if (shortcutButtons && shortcutButtons.length > 0) {
			this.addShortcutButtons(page, m);
		}
```

This check would prevent a crash when `shortcutButtons: undefined` is set in the options

Reply:
Didn't think of that, thanks!